doc/classes/Resource.xml

```diff
@@ -58,8 +73,8 @@
 		<method name="setup_local_to_scene" is_deprecated="true">
 			<return type="void" />
 			<description>
-				Emits the [signal setup_local_to_scene_requested] signal. If [member resource_local_to_scene] is set to [code]true[/code], this method is called from [method PackedScene.instantiate] by the newly duplicated resource within the scene instance.
-				For most resources, this method performs no logic of its own. Custom behavior can be defined by connecting [signal setup_local_to_scene_requested] from a script, [b]not[/b] by overriding this method.
+				Calls [method _setup_local_to_scene]. If [member resource_local_to_scene] is set to [code]true[/code], this method is automatically called from [method PackedScene.instantiate] by the newly duplicated resource within the scene instance.
+				[i]Deprecated.[/i] This method should only be called internally. Define your own methods or override [method _setup_local_to_scene], instead.
```

Comment:
```suggestion
				[i]Deprecated.[/i] This method should only be called internally. Define your own methods or override [method _setup_local_to_scene] instead.
```

Reply:
I attempted to include the original argument of https://github.com/godotengine/godot/pull/67082. In a nutshell, there's no point using this method if you're going to define your own if necessary. It could probably be stripped away because it's... easy to infer.

Reply:
The comma should still be removed.